src/nohttp/suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE suppressions PUBLIC
+	"-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+	"https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+	<suppress checks="io.spring.nohttp.checkstyle.check.NoHttpCheck" files="BeanDefinitionParserDelegate.java" lines="1409"/>
```

Comment:
Do I understand this is referring to an hard-coded line? Couldn't we add something in the code instead?

Reply:
I'm not sure I understand. Can you expand on what you mean by this?

Reply:
I suppose this was meant towards what we discussed earlier today: a marker in the affected Java source file instead of an external reference to a line in a .java file. Even better that we don't need that suppression anymore now.

Reply:
> Even better that we don't need that suppression anymore now.

Was support added for a marker in the source code?

Reply:
Never mind: I just read https://github.com/spring-projects/spring-framework/pull/22839#issuecomment-487636513.